src/declarePropTypesLoosening.tsx

```diff
@@ -0,0 +1,11 @@
+type DeclarePropTypesLooseningType = <TPropTypes>() => <
+  TProps extends TPropTypes
+>(
+  props: TProps,
+) => Omit<TProps, keyof TPropTypes> & TPropTypes
+
+const declarePropTypesLoosening: DeclarePropTypesLooseningType = () => (
```

Comment:
I didn't include `declarePropTypesLoosening()` in the docs because I'm not sure if there's really a use case for it rather than using `declarePropTypesForcing()`